front/tests/pages/operational-studies/scenario-timetable-section.ts

```diff
@@ -373,8 +373,8 @@ class ScenarioTimetableSection extends OpSimulationResultPage {
 
   async getTrainArrivalTime(expectedArrivalTime: string, index = 0) {
     await expect(this.trainArrivalTime.nth(index)).toBeVisible();
-    const actualArrivalTime = await this.trainArrivalTime.nth(index).textContent();
     await expect(this.trainArrivalTimeLoader).toBeHidden();
+    const actualArrivalTime = await this.trainArrivalTime.nth(index).textContent();
     expect(actualArrivalTime).toEqual(expectedArrivalTime);
   }
 
```

Comment:
I recommend using this approach to address the flakiness: https://playwright.dev/docs/test-assertions#expectpoll

```suggestion
  async getTrainArrivalTime(expectedArrivalTime: string, index = 0) {
    await expect(this.trainArrivalTimeLoader).toBeHidden();
    await expect(this.trainArrivalTime.nth(index)).toBeVisible();
    await expect
      .poll(async () => this.trainArrivalTime.nth(index).textContent())
      .toBe(expectedArrivalTime);
  }


```

Reply:
Hmm, I think that just waiting for the loading to finish before reading the time should be enough here. Did you reproduce the flakyness after this change?

Reply:
Yes, I was able to reproduce the flakiness after the change, which is why I recommend using `expect.poll`

Reply:
Done!

Reply:
I think you forgot to push

Reply:
I was waiting for your opinion on the other commit to push, pushed now ^^

I've only kept this one commit.